frontend/src/components/uploadedFileDisplay.module.css

```diff
@@ -0,0 +1,29 @@
+.uploadedFileContainer {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  height: 36px;
+  margin-bottom: 4px;
+  box-sizing: border-box;
+}
+
+.uploadedFile {
+  font-size: 14px;
+  color: var(--grey-900);
+  background-color: var(--grey-100);
```

Comment:
I think the background colour here should be --grey-200

Reply:
Done :)